frappe/desk/reportview.py

```diff
@@ -372,19 +372,76 @@ def delete_report(name):
 @frappe.read_only()
 def export_query():
 	"""export from report builder"""
-	from frappe.desk.utils import get_csv_bytes, pop_csv_params, provide_binary_file
+	from frappe.desk.utils import pop_csv_params
 
 	form_params = get_form_params()
 	form_params["limit_page_length"] = None
 	form_params["as_list"] = True
+	csv_params = pop_csv_params(form_params)
+	export_in_background = int(form_params.pop("export_in_background"))
```

Comment:
```suggestion
	export_in_background = int(form_params.pop("export_in_background"), 0)
```

(test cases are failing because of this being missing / NoneType)